src/lib.rs

```diff
@@ -360,7 +363,8 @@ pub(crate) unsafe fn rav1d_open(c_out: &mut *mut Rav1dContext, s: &Rav1dSettings
             (*f).task_thread.cond = Condvar::new();
             (*f).task_thread.pending_tasks = Default::default();
         }
-        (*f).task_thread.ttd = &mut (*c).task_thread;
+        (&mut (*f).task_thread.ttd as *mut Arc<TaskThreadData>)
+            .write(Arc::clone(&(*c).task_thread));
```

Comment:
Can you just write `.clone()` instead of `Arc::clone`?

Reply:
https://doc.rust-lang.org/std/rc/index.html#cloning-references

Reply:
Hmm, I didn't realize that.  I'm not sure I totally agree with the reasoning (we don't do the same thing for `drop`s of `Arc`s, for example, and we still `#[derive(Clone)]` types containing `Arc`s), but it's okay if it's already a convention.  I haven't been doing that in my existing `Arc` clones, though.

Reply:
No need to go back through and fix anything but I think it's worth doing in the future.